src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -197,20 +196,16 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
        if 'pathlossDb' in sas_response and 'txAntennaGainDbi' in sas_response:
            this_test = (sas_response['pathlossDb'] < ref_response['pathlossDb'] + 1) and (sas_response['txAntennaGainDbi'] < (ref_response['txAntennaGainDbi'] + .2))
 
-           if 'fss' in request:
-               if 'rxAntennaGainRequired' in request['fss']:
-                   if (request['fss']['rxAntennaGainRequired'] == True) :
-                       if 'rxAntennaGainDbi' in sas_response:
-                           this_test = this_test and (sas_response['rxAntennaGainDbi'] < (ref_response['rxAntennaGainDbi'] + .2))
-                       else:
-                           this_test = False
-
+           if 'fss' in request and request['fss']['rxAntennaGainRequired']:
+               if 'rxAntennaGainDbi' in sas_response:
+                   this_test = this_test and (sas_response['rxAntennaGainDbi'] < (ref_response['rxAntennaGainDbi'] + .2))
+               else:
+                   this_test = False
        if this_test:
-           num_passed_tests += 1.0
+           num_passed_tests += 1
        else:
-           num_failed_tests += 1.0
-           
-        if num_failed_tests > max_fail_num: #test fails if number of failed tests greater than allowed
+           num_failed_tests += 1
```

Comment:
I think we want to log the config (and iteration #) so that it's clear which test case failed. This will help tremendously with debugging.

Reply:
Thanks Kate. This has been implemented. Kindly check the latest commit